src/System.IO.IsolatedStorage/src/System.IO.IsolatedStorage.csproj

```diff
@@ -24,34 +26,37 @@
     <Compile Include="System\IO\IsolatedStorage\IsolatedStorageException.Serialization.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetGroup)' != 'uap'">
-    <Compile Include="System\IO\IsolatedStorage\Helper.Win32.Unix.cs" />
+    <Compile Include="System\IO\IsolatedStorage\Helper.Win32Unix.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'uap' AND '$(TargetsWindows)' == 'true'">
+	<Compile Include="System\IO\IsolatedStorage\Helper.Win32.cs" />
+  </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' != 'uap' AND '$(TargetsWindows)' == 'false'">
```

Comment:
This should either be `'$(TargetsUnix)' == 'true'` or `'$(TargetsWindows)' != 'true'`.